lib/puppet/vendor/semantic_puppet/lib/semantic_puppet.rb

```diff
@@ -1,15 +1,7 @@
-begin
-  require 'gettext-setup'
-rescue LoadError
-  def _(msg)
-    msg
-  end
-end
+require 'gettext-setup'
 
 module SemanticPuppet
-  if defined?(GettextSetup)
-    GettextSetup.initialize(File.absolute_path('semantic_puppet/locales', File.dirname(__FILE__)))
-  end
```

Comment:
Yes, but that change, along with dropping the current hard dependency on the gettext gem must be made in the the semantic_puppet gem. It should not be done here. This is just a copy.